draft-ietf-lamps-dilithium-certificates.md

```diff
@@ -465,6 +465,15 @@ even when also exporting the expanded key. ML-DSA seed extraction can be
 implemented by including the random seed xi generated at line 1 of Algorithm 1
 ML-DSA.KeyGen in the returned output.
 
+# Pairwise Consistency Testing
+
+When receiving a private key that contains both the seed and the
+expandedKey, the recipient SHOULD perform a seed consistency check to
+ensure that the sender properly generated the private key.
+
+If the check is done and the seed and the expandedKey are not consistent,
+the recipient MUST reject the private key as malformed.
+
```

Comment:
Note this could be misleading, as NIST and the CMVP use the term PCT to refer only to checking the pairwise consistency of a private key with the corresponding public key. Applying it to "pairwise consistency" of two distinct representations of the private key is novel, and is not covered by the term PCT in FIPS.

The text is mostly fine, but perhaps a different heading (not "Pairwise Consistency Testing") would be more appropriate.
How about "Private Key consistency testing". (Two distinct private keys are not a "key pair" in the usual sense, though they are two keys).

Reply:
I can get behind that.